README.md

```diff
@@ -44,6 +44,7 @@ Add `jquery` to the plugins section of your `.eslintrc` configuration file. You
     "jquery/no-fade": 2,
     "jquery/no-filter": 2,
     "jquery/no-find": 2,
+    "jquery/no-function": 2,
```

Comment:
This should be "no-is-function", and renamed throughout?

Reply:
Will do 👍